vita3k/modules/SceRtc/SceRtcUser.cpp

```diff
@@ -118,55 +97,27 @@ EXPORT(int, sceRtcFormatRFC3339LocalTime) {
 }
 
 EXPORT(int, sceRtcGetCurrentClock, SceDateTime *datePtr, int iTimeZone) {
-    if (datePtr == nullptr) {
-        return RET_ERROR(SCE_RTC_ERROR_INVALID_POINTER);
-    }
-
-    uint64_t tick = rtc_get_ticks(host.kernel.base_tick.tick) + iTimeZone * 60 * 60 * VITA_CLOCKS_PER_SEC;
-    __RtcTicksToPspTime(datePtr, tick);
-
-    return 0;
+    return export_sceRtcGetCurrentClock(host, thread_id, export_name, datePtr, iTimeZone);
 }
 
 EXPORT(int, sceRtcGetCurrentClockLocalTime, SceDateTime *datePtr) {
-    if (datePtr == nullptr) {
-        return RET_ERROR(SCE_RTC_ERROR_INVALID_POINTER);
-    }
-
-    std::time_t t = std::time(nullptr);
-    std::time_t local = std::mktime(std::localtime(&t));
-    std::time_t gmt = std::mktime(std::gmtime(&t));
-    uint64_t tick = rtc_get_ticks(host.kernel.base_tick.tick) + (local - gmt) * VITA_CLOCKS_PER_SEC;
-    __RtcTicksToPspTime(datePtr, tick);
-    return 0;
+    return export_sceRtcGetCurrentClockLocalTime(host, thread_id, export_name, datePtr);
 }
 
 EXPORT(int, sceRtcGetCurrentAdNetworkTick) {
     return UNIMPLEMENTED();
 }
 
 EXPORT(int, sceRtcGetCurrentNetworkTick, SceRtcTick *tick) {
-    if (tick == nullptr) {
-        return RET_ERROR(SCE_RTC_ERROR_INVALID_POINTER);
-    }
-
-    tick->tick = rtc_get_ticks(host.kernel.base_tick.tick);
-
-    return 0;
+    return export_sceRtcGetCurrentNetworkTick(host, thread_id, export_name, tick);
 }
 
 EXPORT(int, sceRtcGetCurrentDebugNetworkTick) {
     return UNIMPLEMENTED();
 }
 
 EXPORT(int, sceRtcGetCurrentTick, SceRtcTick *tick) {
-    if (tick == nullptr) {
-        return RET_ERROR(SCE_RTC_ERROR_INVALID_POINTER);
-    }
-
-    tick->tick = rtc_get_ticks(host.kernel.base_tick.tick);
-
-    return 0;
+    return export_sceRtcGetCurrentTick(host, thread_id, export_name, tick);
```

Comment:
Use the CALL_EXPORT define. this is failling cause you are missing a _ in the name. 
Also export_name is not correct.